tests/Http/Three/SignupTest.php

```diff
@@ -281,10 +281,60 @@ public function testUnauthenticatedUserUpdatingASignup()
         $signup = factory(Signup::class)->create();
 
         $response = $this->patchJson('api/v3/signups/' . $signup->id, [
-            'quantity'     => 888,
             'why_participated'  => 'new why participated',
         ]);
 
         $response->assertStatus(401);
     }
+
+    /**
+     * Test to make sure we are returning quantity correctly.
+     * Quantity is either summed total of quantity across all posts under a signup
+     * or quanttiy under signup if it is still on signup record.
+     *
+     * GET /signups
+     * @return void
+     */
+    public function testQuantityOnSignupIndex()
+    {
+        // Create one signup with a quantity_pending.
+        $firstSignup = factory(Signup::class)->create();
+
+        // Create another signup with a quantity.
+        $secondSignup = factory(Signup::class)->create();
+        $secondSignup->quantity_pending = null;
```

Comment:
I don't think `v3` needs to worry about `quantity_pending` because we will probably kill that after we migrate `quantity` to `posts`.

Reply:
ah true! I'll get rid of this.